charts/vald/values.yaml

```
@@ -3225,3 +3225,232 @@ initializer:
+    # rebalancer.clusterRole.name -- name of clusterRole
+    name: rebalancer
+  # @schema {"name": "rebalancer.clusterRoleBinding", "type": "object"}
+  clusterRoleBinding:
+    # @schema {"name": "rebalancer.clusterRoleBinding.enabled", "type": "boolean"}
+    # rebalancer.clusterRoleBinding.enabled -- creates clusterRoleBinding resource
+    enabled: true
+    # @schema {"name": "rebalancer.clusterRoleBinding.name", "type": "string"}
+    # rebalancer.clusterRoleBinding.name -- name of clusterRoleBinding
+    name: rebalancer
+  # @schema {"name": "rebalancer.serviceAccount", "type": "object"}
+  serviceAccount:
+    # @schema {"name": "rebalancer.serviceAccount.enabled", "type": "boolean"}
+    # rebalancer.serviceAccount.enabled -- creates service account
+    enabled: true
+    # @schema {"name": "rebalancer.serviceAccount.name", "type": "string"}
+    # rebalancer.serviceAccount.name -- name of service account
+    name: vald
+  # @schema {"name": "rebalancer.rebalancer", "type": "object"}
+  rebalancer:
+    # @schema {"name": "rebalancer.rebalancer.rebalancer_job_name", "type": "string"}
+    # rebalancer.rebalancer.rebalancer_job_name -- name of rebalancer job
+    rebalance_job_name: rebalancer-storage-job
+    # @schema {"name": "rebalancer.rebalancer.rebalancer_job_namespace", "type": "string"}
+    # rebalancer.rebalancer.rebalancer_job_namespace -- namespace of rebalancer job
+    rebalance_job_namespance: _MY_POD_NAMESPACE_
+    # @schema {"name": "rebalancer.rebalancer.rebalance_job_template_key", "type": "string"}
+    # rebalancer.rebalancer.rebalance_job_template_key -- template key of rebalancer job
+    rebalance_job_template_key: job.tpl
+    # @schema {"name": "rebalancer.rebalancer.configmap_name", "type": "string"}
+    # rebalancer.rebalancer.configmap_name -- configmap name of rebalancer job
+    configmap_name: rebalancer-storage-job
+    # @schema {"name": "rebalancer.rebalancer.configmap_namespace", "type": "string"}
+    # rebalancer.rebalancer.configmap_namespace -- configmap namespace of rebalancer job
+    configmap_namespace: _MY_POD_NAMESPACE_
+    # @schema {"name": "rebalancer.rebalancer.agent_name", "type": "string"}
+    # rebalancer.rebalancer.agent_name -- name of target agent
+    agent_name: "vald-agent-ngt"
+    # @schema {"name": "rebalancer.rebalancer.agent_namespace", "type": "string"}
+    # rebalancer.rebalancer.agent_namespace -- namespace of target agent
+    agent_namespace: _MY_POD_NAMESPACE_
+    # @schema {"name": "rebalancer.rebalancer.agent_resource_type", "type": "string"}
+    # rebalancer.rebalancer.agent_resource_type -- resource type of target agent
+    agent_resource_type: statefulset
+    # @schema {"name": "rebalancer.rebalancer.reconcile_check_duration", "type": "string"}
+    # rebalancer.rebalancer.reconcile_check_duration -- check duration of rebalancer
+    reconcile_check_duration: 5m
+    # @schema {"name": "rebalancer.rebalancer.tolerance", "type": "float"}
```

```suggestion
    # @schema {"name": "rebalancer.rebalancer.tolerance", "type": "number"}
```
it causes error on linting Helm values.
https://github.com/vdaas/vald/pull/836/checks?check_run_id=1522725098